packages/studio-web/src/app/ras.service.ts

```diff
@@ -46,41 +45,16 @@ export enum SupportedOutputs {
 })
 export class RasService {
   baseURL = environment.apiBaseURL;
-  constructor(private http: HttpClient, private toastr: ToastrService,) {}
+  constructor(private http: HttpClient) {}
 
-  convertRasFormat$(body: ReadAlongFormatRequest, output_type: SupportedOutputs|string): Observable<any|HttpErrorResponse> {
-    return this.http.post(this.baseURL + "/convert_alignment/" + output_type, body, {responseType: 'blob'}).pipe(
-      catchError((err: HttpErrorResponse) => { this.toastr.error(
-        err.message,
-        $localize`Hmm, we can't connect to the ReadAlongs API. Please try again later.`,
-        {
-          timeOut: 60000,
-        }
-      ); return of(err)} ),
-      take(1)
-    );
+  convertRasFormat$(body: ReadAlongFormatRequest, output_type: SupportedOutputs|string): Observable<any> {
+      return this.http.post(this.baseURL + "/convert_alignment/" + output_type, body, {responseType: 'blob'});
```

Comment:
We should keep the error catching in the service, otherwise the code is not DRY and we have to remember to implement the catching of errors anywhere we request the langs. I see you added this logic in the upload component, but it ought to live in the service imo.

Reply:
Yes, I disagree with this.  The error catching *and forward* in the service makes the downstream code *more* complicated, as it has to have a more complicated type signature.  An error is not a piece of data that should be put into the output stream, it's an exception that should be caught at the appropriate place, which, if we are going to do UI stuff to report it to the user, is in the *component* and not in the service.